service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -312,4 +315,15 @@ public void sendEventCreationNotification(String email, String messageBody) {
         String subject = "Notification about event creation status";
         sendEmail(email, subject, messageBody);
     }
+
+    /**
+     * This method validates email address.
+     *
+     * @param emailAddress which will be used for sending letter.
+     */
+    private void validateEmailAddress(String emailAddress) {
```

Comment:
Please move this method to the utility class, I think this could be used as common validation